lib/phobos_db_checkpoint/failure.rb

```diff
@@ -2,6 +2,8 @@ module PhobosDBCheckpoint
   class Failure < ActiveRecord::Base
     include PhobosDBCheckpoint::EventHelper
 
+    scope :ascending_order, -> { order('event_time desc nulls last', 'created_at desc nulls last') }
```

Comment:
the scope is `ascending` but the implementation is `desc`. o.O"
Also, I prefer for the name to mention `time` so that we know what the order is based on, not only the direction. Does it make sense?